Cargo.toml

```diff
@@ -38,7 +39,7 @@ serde_codegen = { version = "0.8", optional = true }
 # Note that not all features are available on all platforms, so even though a
 # feature is enabled some other feature may be used instead.
 [features]
-default = ["libunwind", "libbacktrace", "dladdr", "dbghelp"]
+default = ["libunwind", "libbacktrace", "coresymbolication", "dladdr", "dbghelp"]
```

Comment:
Since it has a fallback I think it could be on by default. If it cannot find the symbols it falls back to dladdr.